vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -139,6 +141,10 @@ void ReactControl::AttachRoot() noexcept {
   auto initialProps = m_initialProps;
   m_reactInstance->AttachMeasuredRootView(m_pParent, std::move(initialProps));
   m_isAttached = true;
+
+#ifdef DEBUG
```

Comment:
In the microsoft fork of RN iOS we changed the if debug stuff to also be available via in release via runtime API call, so supporting development of extensions was easier to do in release builds.  Something we'll probably want in the future...

Reply:
Added TODO

---
In reply to: [309979271](https://github.com/microsoft/react-native-windows/pull/2862#discussion_r309979271) [](ancestors = 309979271)

Reply:
@kmelmon Is there an issue for this? If so, please add a link to it here. Thanks.

Reply:
Opened https://github.com/microsoft/react-native-windows/issues/2870 to track this.